src/com/amazon/corretto/crypto/provider/AesKeyWrapSpi.java

```diff
@@ -44,15 +44,19 @@ final class AesKeyWrapSpi extends CipherSpi {
     private SecretKey jceKey;
     private byte[] keyBytes;
     private int opmode = -1;    // must be set by init(..)
+    private final AccessibleByteArrayOutputStream buffer;
 
     AesKeyWrapSpi(final AmazonCorrettoCryptoProvider provider) {
         Loader.checkNativeLibraryAvailability();
         this.provider = provider;
+        // set initial capacity for the buffer to 0 because we need it to grow
+        // _precisely_ with the amount of data we're buffering.
+        buffer = new AccessibleByteArrayOutputStream(0, Integer.MAX_VALUE);
```

Comment:
This comment doesn't make sense, nor the requirement. It probably makes sense to initialize it to 16 or 32 bytes.

Reply:
this was a change i made while debugging yesterday, i forgot to revisit it. i've reverted to using the default no-arg constructor which initializes the internal buffer to 32 bytes.